### ISSUE TITLE
Running activities are within a frame

### ISSUE THREAD

**Issue by serrq** [jenspfahl/EverydayTasks#41]
To fast determine what task is "running" and what is "pending".

Or in alternative a frame where the "running" tasks are located within.

![1000102592](https://github.com/jenspfahl/EverydayTasks/assets/113377431/1eacc88e-d670-4497-86e4-b7fcb5f7c23c)

**Comment by jenspfahl** [comment]
Isn't the progress bar enough to determine this? What should be the advantage of these led dots?

**Comment by jenspfahl** [comment]
No, paused schedules have a grayed-out progress bar and inactive schedules don't have a progress bar at all. Additionally you can hide inactive schedules within the Sort-menu.

**Comment by jenspfahl** [comment]
Kind of solved with the sort options
